docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -958,50 +979,64 @@ code: |
   add_preview_question = True
 ---
 need:
-  - attachments
-  - attachment_variable_name
   - interview_label
-  - attachment.name
-  - attachment.type
 code: |
-  # This is the question that goes with the regular attachment
   end_question = interview.blocks.appendObject()
   end_question.template_key = 'download screen'
   end_question.data = {"interview_label": interview_label}
   add_end_question = True
----
-need:
-  - attachments
-  - attachment_variable_name
-  - interview_label
-  - attachment.name
-  - attachment.type
+---  
 code: |
-  # TODO(#113): allow multiple attachments here eventually
+  #-----------------------------------------------------------------
+  # Prepare data for output_patterns.yml, each attachment file must provide those 6 attributes.
+  #-----------------------------------------------------------------
+  # Set attributes for the upladed template file     
+  file1 = labels_dict.initializeObject('file1', DAObject)    
+  file1.input_filename = template_upload[0].filename
+  file1.output_filename = interview.file_name
+  file1.attachment_varname = attachment_variable_name    
+  file1.type = template_upload[0].extension  
+  file1.description = interview.description
+  file1.fields = built_in_fields_used + fields + signature_fields            
+      
+  # Set attributes for the "next_steps" docx. 
+  # To add any additional attachment files, follow this example.
+  if interview.form_type != 'other':     
+    file2 = labels_dict.initializeObject('file2', DAObject)    
+    file2.input_filename = 'next_steps_' + interview.form_type + '.docx'
+    file2.output_filename = 'Post-interview-instructions'
+    file2.attachment_varname = file2.output_filename.replace('-', '_')
+    file2.type = 'docx'
+    file2.description = 'next_steps docx file'
+    file2.fields = 'na'
+ 
+  # Bundl-name is a single value
+  if labels_dict.number() > 1:
+    bundle_name = file1.output_filename + '_package.pdf'
+  else:
+    bundle_name = file1.output_filename + '.pdf'
```

Comment:
Nit: this can be simplified:
```suggestion
    file2.fields = 'na'
    bundle_name = file1.output_filename + '_package.pdf'
  else:
    bundle_name = file1.output_filename + '.pdf'
```

Reply:
Well, I think it's better to keep the logic separate from `file2`, so as to leave room for adding other attachment files in the future.